web/index.html

```diff
@@ -1192,7 +1192,7 @@ <h4 class="modal-title" id="switchRegistryModalLabel">Switch Netdata Registry Id
 
 	'disk': {
 		title: 'Disks',
-		info: undefined
+		info: 'All metrics, except space and inodes, are read from <code>/proc/diskstats</code>. The field numbers mentioned in the subsection refer to the field numbers in the <a href="https://www.kernel.org/doc/Documentation/iostats.txt">Kernel Documentation</a>. Space metrics are read via system call <code>statvfs()</code>'
```

Comment:
I suggest the following text:

Charts with performance information for all the system disks. Special care has been given to present disk performance metrics in a way compatible with <code>iostat -x</code>. netdata by default prevents rendering performance charts for individual partitions. Partitions can still be enabled by altering the relative settings in the netdata configuration file.